.github/workflows/long-test.yml

```
@@ -372,17 +306,18 @@ jobs:
         if: success() || failure()
         uses: actions/upload-artifact@v4
         with:
-          name: logs-asan-al
+          name: logs-e2e-release
```

```suggestion
          name: logs-e2e-release-al
```
if we suffix everything, which it looks like we do?